src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
Geoff, even with ```__attribute__((used))``` on the function definition in its own cpp file, it's not getting linked in.  Am I using it incorrectly?